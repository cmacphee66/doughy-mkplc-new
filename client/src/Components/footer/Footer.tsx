import React from "react";
import "./footer.css";

function Footer() {
  return (
    <div className="main-footer">
      <div className="container">
        <div className="row">
          <div className="col">
            <h1 className="list-unstyled">
              Doughy - The Breadlist App
            </h1>
            <h4>Created By - Adrian De Felice - for CodeWorks</h4>
          </div>
          <div className="col">
            <h4>Rework and additional designing by:</h4>
            <ol className="list-unstyled">
              <li>Connor MacPhee</li>
              <li>Wladimir Abdulayev</li>
              <li>Baiju Joseph</li>
            </ol>
          </div>
        </div>
        <hr />
        <div className="row">
          <p className="col-sm">
            &copy;{new Date().getFullYear()} | All rights reserved |
            Terms Of Service | Privacy
          </p>
        </div>
      </div>
    </div>
  );
}

export default Footer;