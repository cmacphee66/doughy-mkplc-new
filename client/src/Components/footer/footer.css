  .main-footer {
      display: flex;
      align-items: flex-end;
      color: white;
      justify-content: space-between;
      background-color: #dc3545;
      padding-top: 3em;
      position: relative;
      bottom: 0;
      width: 100%;
  }