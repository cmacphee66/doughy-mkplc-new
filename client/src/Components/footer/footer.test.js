import React from 'react';
import footer from './Footer'
import renderer from 'react-test-renderer';

const footerTest = describe('footer component', () => {
        it('matches the snapshot for footer', () => {
            const tree = renderer.create(<footer />).toJSON() ;
            expect(tree).toMatchSnapshot()
        })
    })

export default footerTest;
