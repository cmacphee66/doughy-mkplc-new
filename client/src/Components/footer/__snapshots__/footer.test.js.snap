// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`footer component matches the snapshot for footer 1`] = `<footer />`;
